src/zendesk.ts

```diff
@@ -60,8 +60,8 @@ namespace Zendesk {
     group_id?: number
     is_public?: boolean
     tags?: string[]
-    custom_fields?: { id: number; value: any }[]
-    fields?: { id: number; value: any }[]
+    custom_fields?: { id: number; value: string | number | boolean | null | string[] }[]
```

Comment:
Why did you go with this list instead of simply `unknown`? I think it will make casting harder down the line.

Reply:
I thought it would be better to explicility type it since we know all the possible types ahead of time. 
I will switch to `unknown`